### PR TITLE
Add help2man recipe

### PR DESCRIPTION
**Pull request by diorcety**

### CODE REVIEW

File: help2man/help2man.inc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# -*- mode:python; -*-
+RECIPE_TYPES = "machine native"
+
+EXTRA_AUTORECONF="-Iaclocal.m4"
+inherit autotools-autoreconf
```
Comment (esben) [inline help2man/help2man.inc]: Is it necessary to do an autoreconf?

File: help2man/help2man.inc [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# -*- mode:python; -*-
+RECIPE_TYPES = "machine native"
```
Comment (esben) [inline help2man/help2man.inc]: Any special reason for RECIPE_TYPE=machine ?

Comment (diorcety) [inline help2man/help2man.inc]: have help2man in the image? Yes not so usefull :)

Comment (esben) [inline help2man/help2man.inc]: Hehe, let's just change this to RECIPE_TYPES = "native" then :)  That way, we don't have to waste buildbot time building it for all machine archs.

File: help2man/help2man.inc (hunk 1 above)
Comment (esben) [inline help2man/help2man.inc]: Did you try just using "inherit autotools", and did that cause any problems?

Comment (diorcety) [inline help2man/help2man.inc]: can't tell i have to test in on my fedora, centos 6 and ubuntu in order to see if the tools versions are good enough

Comment (esben) [inline help2man/help2man.inc]: Did you find time to test if we could stay with just autotools instead of autotools-autoreconf?

**Pull request merged**